.travis.yml

```diff
@@ -11,6 +11,9 @@ node_js:
   - "3"
   - "4"
   - "5"
+  - "6"
+  - "7"
+  - "8"
```

Comment:
Want to add 9 too?

Reply:
Yeah, why not, I don't have it installed but that's what Travis is for :)